doc/ping.xml

```diff
@@ -118,6 +118,46 @@ xml:id="man.ping">
     IPv6 source routing was deprecated (RFC5095).</para>
   </refsection>
 
+  <refsection>
```

Comment:
BTW, I would've placed the new section after OPTIONS; among the already present miscellaneous info sections like ICMP PACKET DETAILS and so on.